packages/flutter_tools/templates/module/common/lib/main.dart.tmpl

```diff
@@ -18,7 +18,7 @@ class Router extends StatelessWidget {
   Widget build(BuildContext context) {
     final String route = window.defaultRouteName;
     switch (route) {
-      case 'route1':
+      case '/':
         return Container(
           child: Center(child: Text('Route 1\n${packageInfo.appName}')),
```

Comment:
Maybe change this text to say "Default route"?

Reply:
Along this line of thinking, the template for a normal new Flutter app has lots of comments explaining how everything works.

This would be helpful to add here explaining the code as well.